modules/ROOT/pages/salesforce/salesforce-connector-reference-910.adoc

```diff
@@ -0,0 +1,5936 @@
+= Salesforce Connector Reference v9.10 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Release Notes: xref:release-notes::connector/salesforce-connector-release-notes-mule-4.adoc[Salesforce Connector Release Notes]
```

Comment:
I've been adding this text in front of the release notes link: 
For software requirements and compatibility information, see the ....